tutorials/documentation_materials/classiq_101/classiq_concepts/execute/advanced_execution/advanced_execution_user_guide.ipynb

```diff
@@ -0,0 +1,834 @@
+{
```

Comment:
<p>You elaborated regarding the decomposition of the state, but not regarding the decomposition of the Hamiltonian in terms of Paulis. This is the interesting part, where several measurements are needed over non-commutative Paulis.</p><p> </p><p>As a first iteration, we can drop this NOTE and we'll add it later on</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>I also made an effort to improve it in this itreation.</p>